web/cypress/integration/plugin/create-edit-delete-plugin.spec.js

```diff
@@ -36,14 +41,30 @@ context('Create and Delete Plugin List', () => {
     cy.contains('Create').click();
 
     // add test plugins
-    cy.configurePlugins(this.cases);
+    cy.get('@cases').then((cases) => {
+      cy.configurePlugins(cases);
+    })
+  });
+
+  it('should edit the plugin', () => {
+    cy.visit('/plugin/list');
+    cy.get(domSelector.refresh).click();
+    cy.contains(data.name).should('exist').siblings().contains('Edit').click({
+      force: true,
+    });
+    cy.get(domSelector.codemirror)
+      .first()
+      .then(() => {
+        cy.get(domSelector.switch).click();
+        cy.contains('button', 'Submit').click();
+      });
+    cy.contains(data.name).should('not.exist');
   });
 
   it('should delete plugin list', () => {
-    cy.visit('/');
-    cy.contains('Plugin').click();
-    cy.get(domSelectors.refresh).click();
-    cy.get(domSelectors.tableCell).then(function (rows) {
+    cy.visit('/plugin/list');
+    cy.get(domSelector.refresh).click();
+    cy.get(domSelector.tableCell, { timeout }).should('exist').then(function (rows) {
```

Comment:
when I rerun this test locally, I found that sometimes there are some plugins can be  deleted. this will not disturb ci actually,

may be we can try 
`cy.get('.ant-btn-dangerous').should('exist')` instead of `cy.get(domSelector.refresh).click();` to enure that the route list has already shown to make it better ^_^

Reply:
This is a good idea, thanks～

Reply:
![image](https://user-images.githubusercontent.com/72343596/106111323-dc08ac80-6186-11eb-8f76-ed806a5adad4.png)
I used `should()`, but I ran into a CI error again and it seems to be less stable here in the CI test, so I decided to still use `cy.get(domSelector.refresh).click();`

Reply:
got it, thanks